frontend/pages/add_job.vue

```diff
@@ -347,16 +347,126 @@
                             </div>
                         </div>
                         <div class="mt-10 flex flex-col md:flex-row items-center justify-center xl:justify-start">
-                            <NuxtLink to="/options">
-                                <button class="w-full md:w-1/5 ml-0 pt-6 md:pt-0 md:ml-40" type="submit">
+                            <div @click='selectTabOption()' class="w-full md:w-1/5 ml-0 pt-6 md:pt-0 md:ml-40">
+                                <Button show-picture="nothing"
+                                    my-style="background: #235365;padding: 20px 80px;box-shadow: 0 1px 2px rgb(0 0 0 / 0.2);"
+                                    name="Next" />
+                            </div>
+                        </div>
+                    </div>
+                </form>
+
+                <div v-show="isActiveOption === true">
```

Comment:
Hallo @Asam237  did you think of using v-if instead of v-show?

fron definition v-show means the element is available in dom but hide whereas v-if create and delete the element when it value change

Reply:
@Zaker237,
I used v-show to put all the elements in a block

So the elements will be hidden according to the different options